content/runbooks/node/NodeClockNotSynchronising.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Node Clock Not Synchronising
+weight: 20
+---
+
+# NodeClockNotSynchronising
+
+## Meaning
+
+Clock not synchronising.
+
+## Impact
+
+Node instability, apps disconencting.
```

Comment:
```suggestion
Time is not automatically synchronizing on the node. This can cause issues with handling TLS as well as problems with other time-sensitive applications.
```